<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralUI</class>
 <widget class="QWidget" name="GeneralUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="cb_splitter_sizes">
     <property name="text">
      <string>Remember Splitter Sizes</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="cb_colored">
     <property name="text">
      <string>Enable Colored Packets</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="tb_default_dir"/>
     </item>
     <item>
      <widget class="QPushButton" name="bt_default_dir">
       <property name="icon">
        <iconset resource="../icons/icons.qrc">
         <normaloff>:/icons/open.png</normaloff>:/icons/open.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Max Number of Recents</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="tb_recents_num"/>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
